src/constants/creature_defs.lua

```diff
@@ -16,7 +16,7 @@ CREATURE_DEFS = {
     --},
     ['sheep'] = {
         species = 'sheep',
-        grabSound = 'sheepP',
+        grabSound = 'hey-sheep',
```

Comment:
This sound is `sheep-hey`, not `hey-sheep`

Reply:
Oops, never mind - we have this filed as hey-sheep in the sounds.lua table. Carry on